packages/react-i18n/src/errors.ts

```diff
@@ -1,8 +1,8 @@
 import {Replacements} from './types';
 
 export class MissingTranslationError extends Error {
-  constructor(key: string) {
-    super(`Missing translation for key: ${key}`);
+  constructor(key: string, locale?: string) {
+    super(`Missing translation for key: ${key} in locale: ${locale}`);
```

Comment:
Seems weird to include the locale part if locale is undefined?

Reply:
oops, that should not be there. Removing the ?!!!